dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/FacesTest.java

```diff
@@ -0,0 +1,143 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.checkpoint.fat;
+
+import static io.openliberty.checkpoint.fat.FATSuite.getTestMethodNameOnly;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.FileAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestName;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.SkipIfCheckpointNotSupported;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+import componenttest.topology.utils.HttpUtils;
+import io.openliberty.checkpoint.spi.CheckpointPhase;
+
+@RunWith(FATRunner.class)
+@SkipIfCheckpointNotSupported
+public class FacesTest {
+
+    public static final String FRONTEND_SERVER_NAME = "checkpointFaces-frontendUI";
+    public static final String BACKEND_SERVER_NAME = "checkpointFaces-backendServices";
+
+    @Rule
+    public TestName testName = new TestName();
+
+//    @ClassRule
+//    public static RepeatTests repeatTest = MicroProfileActions.repeat(FRONTEND_SERVER_NAME, TestMode.LITE, MicroProfileActions.MP41);
```

Comment:
This is commented out for now.  But shouldn't it include MicroProfileActions.MP50 as the other feature sets to run?

```java
    @ClassRule
    public static RepeatTests repeatTest = MicroProfileActions.repeat("timeoutServer", TestMode.LITE, MicroProfileActions.MP41, MicroProfileActions.MP50);

```

Reply:
Yes.  I can certainly add the mp50 feature set to the MPA repeat action.  I removed it while attempting to debug the baseline mp41 repeat action.  FYI on a relate note, we may need to augment MPA to support feature replacement for multiple servers in a repeat action.